mlrun/api/schemas/artifact.py

```diff
@@ -43,6 +45,18 @@ def to_kinds_filter(self) -> typing.Tuple[typing.List[str], bool]:
             )
 
 
+class ArtifactIdentifier(pydantic.BaseModel):
+    # artifact kind
+    kind: typing.Optional[str]
+    # name actually maps to key in the DB (_find_artifacts for reference), should we rename it?
+    name: typing.Optional[str]
```

Comment:
I think we should change it to `key` since it's what used everywhere else in the API